packages/myst-to-docx/src/utils.ts

```diff
@@ -86,8 +86,14 @@ export function getImageWidth(width?: number | string, maxWidth = MAX_DOCX_IMAGE
   return (lineWidth / 100) * maxWidth;
 }
 
-async function getImageDimensions(file: Blob | Buffer): Promise<{ width: number; height: number }> {
+async function getImageDimensions(
```

Comment:
This is a browser-specific implementation. I don't have the time to gracefully polyfill this on Node.js, but I think that's OK -- it's only used in the web atm anyway, so I make the dimension field a noop on Node.js.